sources/System/GridCentered/GridCenteredSystem.cpp

```diff
@@ -0,0 +1,25 @@
+//
+// Created by Zoe Roux on 5/24/21.
+//
+
+#include "Component/Movable/MovableComponent.hpp"
+#include "Component/GridCentered/GridCenteredComponent.hpp"
+#include "GridCenteredSystem.hpp"
+
+namespace BBM
+{
+	GridCenteredSystem::GridCenteredSystem()
+		: WAL::System({
+			typeid(GridCenteredComponent),
+			typeid(MovableComponent),
+//			typeid(PositionComponent)
```

Comment:
comment ?

Reply:
It is not implemented yet since I am waiting for a real map to test if this grid center thing is a nice addon to movements or not. I am not planning to implement it for this PR but I also don't want to remove something that will be implemented later.